### PR TITLE
On devices page show all events for a device. 

### PR DESCRIPTION
**Pull request by clare**

### CODE REVIEW

File: src/components/Devices/DeviceDetail.vue [hunk 1]
```diff
@@ -1,123 +1,40 @@
...
-    </b-row>
-  </div>
+  <b-container>
+    <b-tabs
+      card
+      class="device-tabs"
+      nav-class="device-tabs-container container"
+      v-model="currentTabIndex"
+    >
+      <b-tab title="Current Software">
```
Comment (gferraro) [inline src/components/Devices/DeviceDetail.vue:9]: Should this and all events have TabTemplate also but just with no value in the badge?

**Review by gferraro — APPROVED** [review]
Looks good, a few minor suggestions.

File: src/components/Devices/DeviceEvents.vue [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+        }
+
+        if (this.date && this.date.length > 0) {
+          this.datePrint = this.date + " 23:59:59";
+          params.endTime = this.datePrint;
+        }
+      }
+
+      this.eventsLoading = true;
+      {
```
[the remark this answers is not in the document]
Comment (clare) [reply src/components/Devices/DeviceEvents.vue:130]: Opps.

File: src/components/Devices/DeviceEvents.vue [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+      eventsLoading: true,
+      events: [],
+      count: 0,
+      page: 1,
+      perPage: 100,
+      eventTypes: [],
+      filtered: false,
+      date: "",
+      datePrint: "",
+      allEventTypes: [
```
[the remark this answers is not in the document]
Comment (clare) [reply src/components/Devices/DeviceEvents.vue:94]: Done :)

**Pull request merged**